dev-python/pulsectl/pulsectl-23.5.2-r1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( pypy3 python3_{9..11} )
```

Comment:
Please remove dead impls from the range when you touch any ebuilds with `PYTHON_COMPAT`, i.e. 9→10 here.